notebook/werewolf.ipynb

```diff
@@ -0,0 +1,489 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "33be6252-0814-4f97-b4ed-06d7e68ce66b",
+   "metadata": {},
+   "source": [
+    "# Werewolf Game"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "722dd171-2a2f-4ed6-b851-8154acc46b83",
+   "metadata": {},
+   "source": [
+    "This is a demo of how to use AgentScope to play the Werewolf game, where six agents play against werewolves and villagers. The details can be found in `examples/werewolf/`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "22684e1a-acf9-469c-8cee-36bd0b3efb8a",
+   "metadata": {},
+   "source": [
+    "To install AgentScope, please follow the steps in [README.md](https://github.com/alibaba/AgentScope/blob/main/README.md#installation)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51b90378-6c48-4462-9110-c44639e57f44",
+   "metadata": {},
+   "source": [
+    "First, we need to set the model configs for the agents."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b221278f-d1b1-410c-892d-df26a0d55fb4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_configs = [\n",
+    "    {\n",
+    "        \"type\": \"openai\",\n",
+    "        \"name\": \"gpt-3.5-turbo\",\n",
+    "        \"api_key\": \"xxx\",\n",
+    "        \"organization\": \"xxx\",\n",
+    "        \"generate_args\": {\n",
+    "            \"temperature\": 0.5\n",
+    "        }\n",
+    "    },\n",
+    "    {\n",
+    "        \"type\": \"post_api\",\n",
+    "        \"name\": \"my_post_api\",\n",
+    "        \"api_url\": \"https://xxx\",\n",
+    "        \"headers\": {\n",
+    "        },\n",
+    "        \"json_args\": {\n",
+    "        }\n",
+    "    }\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b683b8da-38ab-44eb-afc9-9e1a0ab0ec8e",
+   "metadata": {},
+   "source": [
+    "The configs for each agent are as follows. There are six agents, each playing a role in the game. Here, `DictDialogAgent` is a specific class in AgentScope. The details can be found in `src/agentscope/agents/dict_dialog_agent.py`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8a389c49-5474-40db-8b61-e595ec803b82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "agent_configs = [\n",
```

Comment:
The agent configs is too long. It's better to read from file than from raw dicts. Just show a config example here is okay, and omit excessivelly long sentences as follows. 
```
{
    {
        "class": "xxx",
        "args": {
            "name": "Player1",
            "sys_prompt": "Act as a player in a werewolf game. You are Player1 ...", 

        }
    }, 
    # ...
}
```